eyelid-match-ops/src/primitives/poly/modular_poly.rs

```diff
@@ -130,7 +176,7 @@ impl Index<usize> for Poly {
     /// Using `self.coeffs.iter()` is one way to ensure the code only accesses real indexes.
     fn index(&self, index: usize) -> &Self::Output {
         self.coeffs.get(index).expect(
```

Comment:
Only panic if index is above the maximum coefficient:
- in FHE we will have random coefficients, zero in 79 bits is very rare
- we still need to panic if `index` is missing *and* above the maximum coefficient

Reply:
Fixed in [c635f4e](https://github.com/Inversed-Tech/eyelid/pull/33/commits/c635f4e8c74b46321e4dc8f0520ba53271c964f1).

This needed a trick to return a reference to a zero value, because the zero isn't actually in the polynomial coefficient vector.